CHANGELOG.md

```diff
@@ -1,19 +1,25 @@
 # Changelog
 
+## [0.10.1]
+
+### Fixed
+- Changelog does not parse correctly for release #474
+- Directory has wrong permissions in 0.10 #477
```

Comment:
Why this part was not add together with implementation and in separate PR?

Reply:
ok. I see all implemented in this PR

Reply:
See up